.husky/pre-commit

```diff
@@ -0,0 +1 @@
+lint-staged
```

Comment:
I would say, instead add lint to the ci.

Reply:
I've pushed a commit to add it to the CI workflow.

The pre-commit for prettier hook is unobtrusive, as it is fast and won't fail for style changes. If you prefer, I can remove it